File: Modul 7/SOAL 4/Bahasa Python/PRAK604-2210817210028-Muhammad Aulia Rasyid.py
kata1 = input() ; kata2 = input()
isi1 = len(kata1) ; isi2 = len(kata2)
bintang = 0 ; pagar = 0
if isi1 != isi2 :
    print("\nPanjang kalimat berbeda, pesan palsu")
else :
    print()
    for a in range(0, isi1) :
        if kata1[a] == kata2[a] :
            if kata1[a] == ' ' :
                print(" ", end='')
            else :
                print("*", end='') ; bintang += 1
        else :
            print("#", end='') ; pagar += 1
    print("\n\n* =", bintang)
    print("# =", pagar)
    if bintang >= pagar :
        print("\nPesan Asli")
    else :
        print("\nPesan Palsu")

File: Modul 7/SOAL 4/Bahasa C/PRAK604-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>
#include <string.h>

int main (void)
{
    char kata1[100], kata2[100] ;
    int isi1, isi2, a, bintang = 0, pagar = 0 ;
    scanf (" %[^\n]%*c", &kata1) ;
    scanf (" %[^\n]%*c", &kata2) ;
    isi1 = strlen (kata1) ;
    isi2 = strlen (kata2) ;
    if (isi1 != isi2) {
        printf ("\nPanjang kalimat berbeda, pesan palsu\n") ; }
    else {
    printf("\n");
        for (a = 0 ; a < isi1 ; a++) {
            if (kata1[a] == kata2[a]) {
                if (kata1[a] == ' ') {
                    printf (" ") ; }
                else {
                    printf ("*") ;
                    bintang++ ; } }
            else {
                printf ("#") ;
                pagar++ ; } }
        printf ("\n* = %d", bintang) ;
        printf ("\n# = %d", pagar) ;
        if (bintang >= pagar) {
            printf ("\nPesan Asli\n") ; }
        else {
            printf ("\nPesan Palsu\n") ; } }
}
